### DISCUSSION TITLE
Exclamation marks on running miners...

### DISCUSSION THREAD

**Discussion by smyers63** [RainbowMiner/RainbowMiner#2286] [Q&A] [answered]
such as Gminer(!) or Teamblack(!).  What do they mean?

**Comment by RainbowMiner** [comment, marked as answer]
That means either that the GPU/CPU combo has been rented via MiningRigRentals, or that you have setup a `FocusWallet` in pools.config.txt. The (!) emphasizes, that the specific miner is not selected via the profit switching algorithm, but because of one of the above reasons.
